src/Microsoft.TestPlatform.ObjectModel/Microsoft.TestPlatform.ObjectModel.csproj

```diff
@@ -31,31 +31,8 @@
     <ProjectReference Include="..\Microsoft.TestPlatform.CoreUtilities\Microsoft.TestPlatform.CoreUtilities.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="System.Runtime.InteropServices.RuntimeInformation">
-      <Version>4.0.0</Version>
-    </PackageReference>
     <PackageReference Include="System.Reflection.Metadata">
-      <Version>1.3.0</Version>
-    </PackageReference>
-  </ItemGroup>
-  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.4' ">
-    <PackageReference Include="System.Runtime.Serialization.Primitives">
-      <Version>4.1.1</Version>
-    </PackageReference>
-    <PackageReference Include="System.Xml.XPath.XmlDocument">
-      <Version>4.0.1</Version>
-    </PackageReference>
-    <PackageReference Include="System.ComponentModel.EventBasedAsync">
-      <Version>4.0.11</Version>
-    </PackageReference>
-    <PackageReference Include="System.ComponentModel.TypeConverter">
-      <Version>4.1.0</Version>
-    </PackageReference>
-    <PackageReference Include="System.Runtime.Serialization.Json">
-      <Version>4.0.2</Version>
-    </PackageReference>
-    <PackageReference Include="System.Reflection.TypeExtensions">
-      <Version>4.1.0</Version>
+      <Version>1.6.0</Version>
```

Comment:
>1.6.0 [](start = 15, length = 5)

Do we bump up this version or remain at 1.3.0?

Reply:
@singhsarab

Reply:
> > �
> 
> ?? Please check this
> 
> Refers to: test/TestAssets/SimpleTestProjectx86/SimpleTestProjectx86.csproj:12 in [4964d3b](https://github.com/microsoft/vstest/commit/4964d3b2e19777489de8afcd2b3444fa4d180229). [](commit_id = [4964d3b](https://github.com/microsoft/vstest/commit/4964d3b2e19777489de8afcd2b3444fa4d180229), deletion_comment = False)

Seems normal when opened in VS.

Reply:
Keep it at 1.3.0 if it has netstandard2.0 support

---
In reply to: [301028228](https://github.com/microsoft/vstest/pull/2067#discussion_r301028228) [](ancestors = 301028228)